front-end/sass/themes/color-stages.sass

```diff
@@ -103,34 +108,36 @@ $color-stages-gutter: 5px
     .status__detail-stage,
     .status__detail-job
         flex: 1
-        margin-left: $color-stages-gutter
+        margin: $color-stages-gutter
```

Comment:
The space is now not equal between the stages, there is too much space between the jobs and stages, don't you think?

![image](https://user-images.githubusercontent.com/6495166/50640234-81c79e00-0f64-11e9-9088-fb8d9dbfad6a.png)

Reply:
I don't know how to show the stages, i only get jobs

Reply:
Run `make cypress-run`, that should fill your dashboard with test-data. You can kill the script half-way trough the pipeline, so you see both jobs and stages

Reply:
And how do i kill the script? Is that by doing CTRL + C or something else?

Reply:
Not sure how to do that on an apple but yes, CTRL + C on linux/windows

Reply:
Thanks, that will be CMD + C on Mac

Reply:
Why is there at the image above no boxshadow around the status__detail